src/renderer.ts

```diff
@@ -125,7 +129,13 @@ const renderSlide = async (
 ) => {
   slide.hidden = node.hidden;
   if (node.backgroundColor) slide.background = { fill: node.backgroundColor };
-  if (node.backgroundImage) slide.background = { path: node.backgroundImage };
+  if (node.backgroundImage) {
+    if (typeof node.backgroundImage === 'string') {
```

Comment:
This fallback will probably not be needed if we go for the full API breaking change

Reply:
👍